### ISSUE TITLE
Jaws 2022 read `Form region` for expanded/collapsed buttons

### ISSUE THREAD

**Issue by ThaMoreira** [FreedomScientific/VFO-standards-support#681]
### Summary
 1. Go to http://snippix.only.sap/snippets/480523
 2. Use Jaws Screen reader to read the expanded/collapsed button

### Expected result

It is expected that Jaws does not read "Form Region" for the button, just the the button name and additional information

### Actual result

Jaws is reading "Form region" for the button, as seen bellow.

![image](https://user-images.githubusercontent.com/36004615/202256726-3665401f-a5ed-44f2-9006-55c02a505dae.png)

> A test case that demonstrates the issue - http://snippix.only.sap/snippets/480523

#### JAWS version 
jaws 2022

**Comment by JAWS-test** [comment]
Your link does not work. The output of the regions by the screen reader is normal and desired behavior when entering the regions.

**Comment by ThaMoreira** [comment]
The link works fine for me, I opened even on a Incognito Window (http://snippix.only.sap/#480523), after the page is fully loaded you need to press the play button to execute the script. Could you please try again?

**Comment by JAWS-test** [comment]
> The link works fine for me

Probably because you are in the SAP network. The page is not accessible from outside.

**Comment by ThaMoreira** [comment]
Sorry I did not pay attention on this, but if the role is "Form" Jaws should announce "Form Region" ?

Snippet example: https://formregion.w3spaces.com/form-region.html

**Comment by JAWS-test** [comment]
You are using a `<div>` with `role=form`. According to [Core AAM](https://www.w3.org/TR/core-aam-1.2/), the mapping distinguishes between role=form with and without label, but Chrome submits form to the API (F12 > Accessibility). I don't see any form region in Accessibility Insights. Whether this is actually a JAWS bug - no idea. In itself, the output of the regions when entering is correct, but here there should be no region according to Core AAM (due to missing label). Maybe also a bug of the browser?

NVDA outputs the form region as well. Also in the element overview of NVDA, which JAWS does not do. In linear reading, both screen readers do not output the form region.

What helps:
* remove role=form
* use `<form>` if needed (will not be output as region)
